openapi/generator_test.go

```diff
@@ -260,6 +295,12 @@ func TestNewSchemaFromStructFieldErrors(t *testing.T) {
 	assert.Len(t, g.Errors(), 4)
 	assert.NotEmpty(t, g.Errors()[2].Error())
 	assert.NotEmpty(t, g.Errors()[3].Error())
+
+	// Field D has example value that cannot be parsed to bool.
+	sor = g.newSchemaFromStructField(typ.Field(3), false, "D", typ)
+	assert.NotNil(t, sor)
+	assert.Len(t, g.Errors(), 5)
+	assert.NotEmpty(t, g.Errors()[4].Error())
```

Comment:
Testing that the error message is not empty is good, but if we're doing that, we could at least test for the presence of other fields as-well (fiedl name, type...) if that's relevant to render a proper/understandable error message.

Reply:
I was just copying this from the test above 🙈  But yes, we should do more. Changed, so we now assert the error message.